.clang-tidy

```diff
@@ -26,6 +26,7 @@ cppcoreguidelines-*,
 -cppcoreguidelines-pro-type-member-init,
 -cppcoreguidelines-pro-bounds-pointer-arithmetic,
 -cppcoreguidelines-pro-type-reinterpret-cast,
+-cppcoreguidelines-pro-type-union-access,
```

Comment:
To access `msgpack` object per type we need to use the `via` attribute which is a union. `clang-tidy` does not like union. But we need union in this case.

Reply:
fair point. maybe you can restrict this usage to just that line using `// NOLINT(cppcoreguidelines-pro-type-union-access)` (or `NOLINTNEXTLINE` if the line gets too long otherwise)? That way, it becomes very clear that there's only one occurrence and it has is intentional (e.g.: when refactoring that line, you can see something like "the latest version now _does_ support it so we can refactor that"

Reply:
Do you know how to ignore one line in clang-tidy?

Reply:
yes you can do either

```cpp
// NOLINTNEXTLINE(cppcoreguidelines-pro-type-union-access)
foo = bar;
```

or

```cpp
foo = bar;  // NOLINT(cppcoreguidelines-pro-type-union-access)
```

Reply:
done